src/docs/resources/faq.md

```diff
@@ -126,14 +126,14 @@ and address commonly-requested user features.
 
 Flutter includes:
 
-* Heavily optimized, mobile-first 2D rendering engine
+* Heavily optimized, the mobile-first 2D rendering engine
   with excellent support for text
 * Modern react-style framework
 * Rich set of widgets implementing Material Design and iOS-style
 * APIs for unit and integration tests
 * Interop and plugin APIs to connect to the system and 3rd-party SDKs
 * Headless test runner for running tests on Windows, Linux, and Mac
-* [Dart DevTools][] for testing, debugging, and profiling your app
+* [Dart DevTools][] for testing, debugging and profiling your app
```

Comment:
At Google, we use the Oxford (or serial) comma!!  https://developers.google.com/style/commas?hl=en#serial-commas